addons/hr/hr.py

```diff
@@ -247,10 +247,11 @@ def onchange_department_id(self, cr, uid, ids, department_id, context=None):
         return {'value': value}
 
     def onchange_user(self, cr, uid, ids, user_id, context=None):
-        work_email = False
         if user_id:
-            work_email = self.pool.get('res.users').browse(cr, uid, user_id, context=context).email
-        return {'value': {'work_email': work_email}}
+            user = self.pool['res.users'].browse(cr, uid, user_id, context=context)
+            return {'value': {'name': user.name, 'work_email': user.email, 'image_medium': user.image_medium}}
```

Comment:
As specified by @odony in the other PR, the dict returned with the data would be much better split into multiple indented lines: https://github.com/odoo/odoo/blob/8.0/addons/account/account_invoice.py#L465-L469

As specified by @tde-banana-odoo , you must set user.image, because image_medium is a function field: https://github.com/odoo/odoo/blob/8.0/addons/hr/hr.py#L216